src/canister_tests/Cargo.toml

```diff
@@ -14,4 +15,5 @@ internet_identity_interface = { path = "../internet_identity_interface" }
 candid = "0.7"
 ic-cdk = "0.5"
 ic-types = "0.3"
+ic-error-types = { git = "https://github.com/dfinity/ic", rev = "46b3de985fef2cc9e326ed191c266b610512443b" }
```

Comment:
I _think_ that if you just add `ic-error-types = "*"` it will pull it from the git checkout and then there's no risk of updating one rev but not the other